qutip/floquet.py

```diff
@@ -176,7 +186,9 @@ def floquet_modes_t(f_modes_0, f_energies, t, H, T, args=None):
 
     return f_modes_t
 
-def floquet_modes_table(f_modes_0, f_energies, tlist, H, T, args=None):
+
+def floquet_modes_table(f_modes_0, f_energies, tlist, H, T, args=None,
+                        options=Options()):
```

Comment:
`Options()` is a mutable object, so shouldn't be used as a function default.  You can instead do
```python
def floquet_modes_table(..., options=None):
    options = options or Options()
    ...
```